test/matrix/CMakeLists.txt

```
@@ -1 +1,2 @@
+ginkgo_create_common_test(csr_kernels)
 ginkgo_create_common_test(dense_kernels)
```

nit: I don't think the current contributing guidelines contain something on a new line at the end of a file, but I think it would be good to use it by default
```suggestion
ginkgo_create_common_test(dense_kernels)

```

---

I think it is also be handled by editorconfig